terraform/examples/daos_cluster/terraform.tfvars.perf.example

```diff
@@ -26,11 +26,34 @@ server_daos_scm_size       = 45
 # server_os_disk_type        = "pd-ssd"
 
 # pools = [
-#  {
-#    pool_name = "test_pool"
-#    pool_size = "1TB"
-#    containers = [ "test_container1", "test_container2" ]
-#  }
+#   {
+#     name = "test_pool",
+#     size = "1TB",
+#     tier_ratio = 3
+#     acls = [
+#       "A::OWNER@:rwdtTaAo",
+#       "A:G:GROUP@:rwtT"
```

Comment:
It would be better to show a actual user name in this place.  It is not clear to me where to add my user name.

Reply:
Maybe a comment with example would be better?